operations/helm/charts/mimir-distributed/ci/test-oss-multizone-values.yaml

```diff
@@ -6,11 +6,6 @@ ci_resources: &ci_resources
   requests:
   limits:
 
-configStorageType: Secret
-
-enterprise:
-  enabled: true
-
```

Comment:
It is. We discussed writing static tests for it though, instead of `ct` tests. I'll create an issue to track it if you agree.

Proposed issue: verify in static test that configuration is correctly encoded in a secret and is mounted on the proper path in every mimir/gem component.

Reply:
sounds good. I am happy to help with that if needed because rego seemed non-trivial to me.

Reply:
moved to new kind of golden record after all